vendor/engines/badges_engine/lib/badges_engine.rb

```diff
@@ -0,0 +1,4 @@
+require "badges_engine/engine"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.